app/Http/Controllers/Daemon/ActionController.php

```diff
@@ -51,8 +51,10 @@ public function markInstall(Request $request)
             ], 403);
         }
 
-        $server->installed = ($status === 'installed') ? 1 : 2;
-        $server->save();
+        Server::where('id', $server->id)
```

Comment:
This change breaks the rest of the code pathway because the installed state is never set on the existing model. I'm not sure what is leading you to believe this is broken, I've had it work fine in every test I've ever done with it?

Reply:
I should clarify, finding the actual cause of the failure for it to persist to the database would be more ideal than just changing the code here.